jekyll/_cci2/android-machine-image.adoc

```diff
@@ -1,51 +1,44 @@
 ---
```

Comment:
just an observation: it seems odd that this file is showing as an edited file, whereas the other conversions are shown as entirely new files 🤔